src/storage/DataAccess.js

```diff
@@ -32,6 +32,13 @@
  *   @param {string} collectionName - name of collection to delete from
  *   @param {function(object): boolean} whereFn - function to identify rows to delete
  *   @return {void}
+ *
+ * Insert or Update:
+ * - upsert(collectionName, value, whereFn)
+ *   @param {string} collectionName - name of the collection to add to
+ *   @param {object|array} value - object to add, or array of objects to add
```

Comment:
I think we probably want this to just be `object` - not sure how the update operation would work given an array

Reply:
Agreed. I had copied this from the insert comments without even thinking about it